custom/plugins/VirtuaTechnology/Resources/menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/menu.xsd">
+    <entries>
+        <entry>
```

Comment:
Form task description: add a link in backend -> Items tab add Virtua section and Technologies link
Now you have Items -> VirtuaTechnologies should be Items->Virtua->Technologies